lib/omnibus/packagers/rpm.rb

```diff
@@ -495,8 +492,8 @@ def create_rpm_file(fips, debug = false)
           # Generate a temporary home directory
           home = Dir.mktmpdir
         end
-
-        if fips
+        recent_version = rpm_version()[:minor].to_i >= 14 ? true : false
```

Comment:
Can we make the name more descriptive? Seeing a variable called `recent_version` doesn't tell you much when you see it. I'd suggest something like `rpm_414_or_later`

Reply:
Sure